homeassistant/components/fronius/quality_scale.yaml

```diff
@@ -0,0 +1,87 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  appropriate-polling: done
+  brands: done
+  common-modules:
+    status: done
+    comment: |
+      Single platform only, so no entity.py file.
+      CoordinatorEntity is used.
+  config-flow-test-coverage: done
```

Comment:
`test_reconfigure_already_configured` stale test name, as we are trying to reconfigure a different device

Reply:
`test_reconfigure_already_existing` I am not sure if this test is relevant though

Reply:
#132872